docs/index.rst

```diff
@@ -32,6 +32,7 @@ Welcome to Cowrie's documentation!
    supervisor/README.rst
    systemd/README.rst
    threatjammer/README.rst
+   datadog/README.rst
```

Comment:
this list is organized by alphabetical order, so we probably need to bump Datadog up to respect it

Reply:
You're right!